src/UserManager.ts

```diff
@@ -354,8 +356,7 @@ export class UserManager {
 
             logger.info("success, user not authenticated");
             return null;
-        }
-        catch (err) {
+        } catch (err) {
```

Comment:
please avoid changing code style,

Reply:
I am sorry, this is something that my IDE does if not otherwise configured. Perhaps you should consider enforcing this specific rule as well via the .editorconfig file if possible.